src/app/components/home-page/home-page.component.ts

```diff
@@ -47,6 +52,14 @@ export class HomePageComponent implements OnInit, OnDestroy {
         })
       );
     }
+    if (this._isAuth$.value && !this._userStatistic) {
+      this._subscriptions.add(
+        this.statisticService.getUserStatistic().subscribe(statistic => {
```

Comment:
I would suggest creating const called sub and assigning everything inside Add to tit this.statisticService.getUserStatistic().subscribe(statistic => {
          statistic = statistic || [];
          this.statisticService.setStatistic(statistic);
        }

and then adding sub variable to subscriptions array. This will make code more readable.